cves/kernel/CVE-2013-2094.yml

```diff
@@ -116,362 +29,104 @@ upvotes_instructions: |
   upvotes score on your branch.
 upvotes:
```

Comment:
I'll add 2 points to this, not super interesting as a vulnerability but privilege escalation is always cool